src/kf_scripts/change_dates.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
Is there a reason that we aren't reading the entire yaml file in and then writing it out? Using `startswith` seems a bit error prone?